test/cluster/types.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiPromise } from "@polkadot/api";
```

Comment:
maybe move this to utils? 
I think multinode deserves an special subfoldering like UI tests :)